### ISSUE TITLE
Not using directory-level .eslintrc

### ISSUE THREAD

**Issue by shellscape** [AtomLinter/linter-eslint#988]
## Issue Type
Probable Bug

## Issue Description
linter-eslint doesn't appear to be using the .eslintrc files local to a particular directory. This is useful when different directories within the same project have different linting needs. When linter-eslint runs against a file in a directory with it's own .eslintrc file, it incorrectly reports errors based on rules from the .eslintrc file in the project root. I've run ESLint via the command line where it respects nested or directory-level .eslintrc files and the result is different. Screenshots for reference:

![atom](http://i.imgur.com/1g6NnsO.png)

![terminal](http://i.imgur.com/dPkcDnL.jpg)

## Bug Checklist
- [done] Restart Atom
- [verified] Verify the `eslint` CLI gives the proper result, while `linter-eslint` does not
- [done] Paste the output of the `Linter Eslint: Debug` command from the Command Palette below

```
Atom version: 1.19.1
linter-eslint version: 8.2.1
ESLint version: 3.19.0
Hours since last Atom restart: 22.6
Platform: darwin
Using local project ESLint from: /github/ui-tracking/node_modules/eslint
Current file's scopes: [
  "source.js",
  "meta.method-call.js",
  "meta.arguments.js",
  "string.quoted.single.js"
]
linter-eslint configuration: {
  "lintHtmlFiles": false,
  "useGlobalEslint": false,
  "showRuleIdInMessage": true,
  "disableWhenNoEslintConfig": true,
  "eslintrcPath": "",
  "globalNodePath": "",
  "advancedLocalNodeModules": "",
  "eslintRulesDir": "",
  "disableEslintIgnore": false,
  "disableFSCache": false,
  "fixOnSave": false,
  "scopes": [
    "source.js",
    "source.jsx",
    "source.js.jsx",
    "source.babel",
    "source.js-semantic"
  ],
  "rulesToSilenceWhileTyping": [],
  "rulesToDisableWhileFixing": []
}
```

**Comment by IanVS** [comment]
This definitely works in most cases, so I think you must have hit some kind of edge case.  Is it possible to share a small repo that reproduces the problem so that we can investigate further?  It looks like you have a `package.json` in the same folder as `version-conflicts.js`, so I'm guessing it might have something to do with that...

**Comment by shellscape** [comment]
@IanVS I agree, it's likely an edge case. The repo this is happening in isn't public (yet). I'll rename that package.json, as it's just an informational stub in that directory, and see if that has any effect on the issue. Please bear with me, it might take me a bit to circle back to it.

**Comment by Arcanemagus** [comment]
@shellscape Did you get a chance to determine a way to reproduce this that you can share?

**Comment by Arcanemagus** [comment]
Marking this as closed due to a lack of response, happy to reopen this if you find a way to reproduce it though!
